<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wrench</class>
 <widget class="QWidget" name="Wrench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wrench</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>待上传信息表</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="infText"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="pageSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="pageLabel">
         <property name="text">
          <string>0/0页</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="searchWrenchBt">
         <property name="text">
          <string>搜索扳手</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="numofWrechLb">
         <property name="text">
          <string>扳手数目：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wrenchLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="setCountBt">
         <property name="text">
          <string>设置计数值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wrenchIdLb">
         <property name="text">
          <string>扳手ID：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="wrechSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="countLb">
         <property name="text">
          <string>计数值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="CountspinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
